src/commands/commander.h

```diff
@@ -67,6 +67,31 @@ enum CommandFlags : uint64_t {
   kCmdNoDBSizeCheck = 1ULL << 12,  // "no-dbsize-check" flag
 };
 
+enum class CommandCategory : uint8_t {
+  Unknown = 0,
+  Bit,
+  BloomFilter,
+  Cluster,
+  Function,
+  Geo,
+  Hash,
+  HLL,
+  JSON,
+  Key,
```

Comment:
"Key" is weird here since most methods would have a "Key"?

Reply:
Key is for key-related commands, same as in redis.